lib/shared/util.ts

```diff
@@ -0,0 +1,13 @@
+const nodeUtil = require("util");
```

Comment:
I'm not sure if we can do it everywhere
![image](https://user-images.githubusercontent.com/5860190/29446674-26992a62-83f6-11e7-8cab-b64f20862360.png)